app/views/orders/payment.html.erb

```diff
@@ -0,0 +1,52 @@
+<section>
+
+  <% if @order.errors.any? %>
+    <ul class="errors">
+      <% @order.errors.each do |column, message| %>
+        <li>
+          <strong><%= column.capitalize %></strong> <%= message %>
+        </li>
+      <% end %>
+    </ul>
+  <% end %>
+
+  <%= form_with model: @order,url: complete_path(@order), method: :patch, class: 'order-form' do |f| %>
```

Comment:
Maha requested feedback: I see that the class `order-form` is applied to the payment form, but I don't see any styles that correspond to this class. I believe this is from boostrap, but it seems that the rest of your html doesn't make use of bootstrap classes to make a justified form. I would recommend looking at a few more examples like this: https://mdbootstrap.com/snippets/jquery/pjoter-2-0/747937